Tools/TestWebKitAPI/Tests/WebCore/SharedBuffer.cpp

```diff
@@ -80,20 +81,20 @@ TEST_F(FragmentedSharedBufferTest, appendBufferCreatedWithContentsOfExistingFile
     ASSERT_NOT_NULL(buffer);
     SharedBufferBuilder builder;
     builder.append(*buffer);
-    builder.append("a", 1);
+    builder.append("a"_span);
     EXPECT_TRUE(builder.size() == (strlen(FragmentedSharedBufferTest::testData()) + 1));
     EXPECT_TRUE(!memcmp(builder.get()->makeContiguous()->data(), FragmentedSharedBufferTest::testData(), strlen(FragmentedSharedBufferTest::testData())));
     EXPECT_EQ('a', builder.get()->makeContiguous()->data()[strlen(FragmentedSharedBufferTest::testData())]);
 }
 
 TEST_F(FragmentedSharedBufferTest, tryCreateArrayBuffer)
 {
-    char testData0[] = "Hello";
-    char testData1[] = "World";
-    char testData2[] = "Goodbye";
-    SharedBufferBuilder builder(std::in_place, testData0, strlen(testData0));
-    builder.append(testData1, strlen(testData1));
-    builder.append(testData2, strlen(testData2));
+    const auto testData0 = "Hello"_span;
+    const auto testData1 = "World"_span;
+    const auto testData2 = "Goodbye"_span;
```

Comment:
`constexpr`? Or don’t use local variables?

Reply:
`_span` doesn't currently work with constexpr. It relies on bitwise_cast() which calls memcpy() which is not allowed in constexpr context.